README.md

```diff
@@ -200,6 +200,7 @@ Options|Default|Description
 ------|------|------
 \-\-bg-color=&lt;R,G,B&gt;|0.2, 0.2, 0.2|Set the window *background color*.<br>Ignored if *hdri* is set.
 \-\-resolution=&lt;width,height&gt;|1000, 600|Set the *window resolution*.
+\-\-position=&lt;x,y&gt;||Set the *window position* (top left corner) , in pixels, starting from the top left of your screens.
```

Comment:
Could you validate that on MacOS @Meakk ?

Reply:
It's incorrect: bottom left corner of the window, bottom left of the screen.

Reply:
Should be fixed in VTK imo.

Reply:
Is there any issue already opened in VTK?

Reply:
None that I can find, ill create it

Reply:
Some resource:

Xorg: https://tronche.com/gui/x/xlib/introduction/overview.html : upper left
Windows: http://www.jasinskionline.com/windowsapi/ref/s/setwindowpos.html: Upper left
Cocoa: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CocoaDrawingGuide/Transforms/Transforms.html: Lower left

Reply:
SDL: https://wiki.libsdl.org/SDL_SetWindowPosition upper left

Reply:
https://gitlab.kitware.com/vtk/vtk/-/issues/18681

Reply:
So this may take a while. What about we handle that application side for now ?